src/lib/components/dropdown/Dropdown.component.js

```diff
@@ -101,86 +99,125 @@ const Caret = styled.span`
 
 const TriggerStyled = ButtonStyled.withComponent("div");
 
+const DropdownTriggerContainer = forwardRef<DropdownTriggerContainerProps, Element>(({isItem, dataIndex, open, size , variant, title, onBlur, onFocus, onClick, onMouseEnter, onMouseLeave, children, ...rest}, ref) => {
```

Comment:
I am wondering do you have the prettier extension installed? Please enable it to make sure the code follows the `.prettierrc` config.
Thanks!

Reply:
done